### PR TITLE
Fixed Doxygen

### PR DESCRIPTION
**Pull request by gabryelreyes**
Warnings on undocumented members were silenced by Doxyfile option

### CODE REVIEW

File: lib/APPRemoteControl/SerialMuxChannels.h [hunk 1]
```diff
@@ -76,26 +76,26 @@
...
-    uint8_t response;
+    uint8_t response; /**< Response to the command */
 } __attribute__((packed)) CommandResponse;
 
 /** Struct of the "Speed" channel payload. */
 typedef struct _SpeedData
 {
-    int16_t left;
-    int16_t right;
+    int16_t left;  /**< Left motor speed */
```
Comment (BlueAndi) [inline lib/APPRemoteControl/SerialMuxChannels.h:91]: Missing unit

File: lib/APPRemoteControl/SerialMuxChannels.h [hunk 2]
```diff
@@ -76,26 +76,26 @@
...
+    int16_t right; /**< Right motor speed */
```
Comment (BlueAndi) [inline lib/APPRemoteControl/SerialMuxChannels.h:92]: Missing unit

File: lib/APPRemoteControl/SerialMuxChannels.h [hunk 3]
```diff
@@ -76,26 +76,26 @@
...
 } __attribute__((packed)) SpeedData;
 
 /** Struct of the "Line Sensor" channel payload. */
 typedef struct _LineSensorData
 {
-    uint16_t lineSensorData[5U];
+    uint16_t lineSensorData[5U]; /**< Line sensor data */
```
Comment (BlueAndi) [inline lib/APPRemoteControl/SerialMuxChannels.h:98]: Missing unit ... I guess the values in [digits] and normalized right?

File: lib/Service/Logging.h [hunk 4]
```diff
@@ -41,24 +41,29 @@
  *****************************************************************************/
 
 #ifndef LOG_FATAL_ENABLE
+/** Enable/disable fatal log messages. */
 #define LOG_FATAL_ENABLE (1)
-#endif /* LOG_FATAL_ENABLE */
```
Comment (BlueAndi) [inline lib/Service/Logging.h:45]: Why was this removed? It was style guide.

File: lib/Service/Logging.h [hunk 5]
```diff
@@ -183,10 +208,15 @@
...
 #if (0 == LOG_DEBUG_ENABLE)
 
-#define LOG_DEBUG(_filename, _msg)           (void)(_filename)
+/** Log debug message. */
+#define LOG_DEBUG(_filename, _msg) (void)(_filename)
+/** Log debug message with additional value. */
 #define LOG_DEBUG_VAL(_filename, _msg, _val) (void)(_filename)
-#define LOG_DEBUG_HEAD(_filename)            (void)(_filename)
+/** Log debug header. */
+#define LOG_DEBUG_HEAD(_filename) (void)(_filename)
```
Comment (BlueAndi) [inline lib/Service/Logging.h:216]: Why we keep here (void)(_filename)?

**Review by BlueAndi — CHANGES_REQUESTED** [review]

File: lib/Service/Logging.h (hunk 5 above)
Comment (gabryelreyes) [reply lib/Service/Logging.h:216]: When DEBUG is not enabled, the TAGs in several modules will cause the compilation process to fail as these are not used.

Comment (gabryelreyes) [reply lib/Service/Logging.h:216]: See https://github.com/BlueAndi/RadonUlzer/actions/runs/6810581334/job/18519155506

Comment (BlueAndi) [reply lib/Service/Logging.h:216]: Move the enable flags to the pio env's. Remove all LOGs from HAL + Service.

**Review by BlueAndi — CHANGES_REQUESTED** [review]

**Review by BlueAndi — APPROVED** [review]
